### DISCUSSION TITLE
请问如何一键安装历史版本，我查看历史都是文件，没有一键安装的按钮

### DISCUSSION THREAD

**Discussion by DX3901** [the1812/Bilibili-Evolved#3291] [提问 / Q&A] [answered]
我查看历史都是文件，没有一键安装的按钮

**Comment by sky-279** [comment, marked as answer]
你可以试试下载历史版本release，然后在油猴中手动导入disk文件夹下bilibili-evolved.user.js文件

**Reply by Geeyun-JY3** [reply]
Releases 中的 zip 文件版本不是对应的。

**Reply by DX3901** [reply]
谢谢大家的回答

**Comment by Geeyun-JY3** [comment]
其实是有直链的，调整 URL 中的版本号即可，不过实测和 Release 类似，URL 中的版本和实际版本好像也是不对应的，如：https://cdn.jsdelivr.net/gh/the1812/Bilibili-Evolved@v2.1.7/bilibili-evolved.user.js
